test/integration/helpers.js

```diff
@@ -92,3 +96,31 @@ exports.waitFor = function(item, timeout, msg) {
   msg = msg || '';
   return browser.wait(item, timeout, msg);
 };
+
```

Comment:
I like that we can reuse this.  Maybe we should do a `helpers/` directory & put this in `helpers/projectHelpers.js`